python/deltalake/table.py

```diff
@@ -210,7 +211,22 @@ def metadata(self) -> Metadata:
         """
         return self._metadata
 
-    def vacuum(self, retention_hours: Optional[int] = None, dry_run: bool = True) -> List[str]:
+    def history(self, limit: Optional[int] = None) -> List[CommitInfo]:
+        """
+        Run the history command on the DeltaTable.
+        The operations are returned in reverse chronological order.
+
+        :param limit: the commit info limit to return
+        :return: list of the commit infos registered in the transaction log
+        """
+        return [
+            CommitInfo(commit_info_raw=commit_info_raw)
```

Comment:
I think we can further simplify the code by returning a list of python dictionaries here instead of `CommitInfo` objects. Given commit info is a very dynamic json dictionary that contains arbitrary keys, we don't get much value in creating a specific `CommitInfo` type here.